packages/asset-server-plugin/src/plugin.ts

```diff
@@ -232,6 +232,20 @@ export class AssetServerPlugin implements NestModule, OnApplicationBootstrap {
                 }
                 res.contentType(mimeType);
                 res.setHeader('content-security-policy', `default-src 'self'`);
+                // Configure Cache-Control header
```

Comment:
I think we should make all of this logic run once on startup, and save the resulting cache header string to a private class field, otherwise it is being run on every single asset request.

You could argue this is a micro-optimization, but I also think it'll make for more readable code.